site/ui/_animationUtils.js

```diff
@@ -41,10 +41,10 @@ export const CHROME_HORIZONTAL_PADDING = 40;
 export const CHROME_VERTICAL_PADDING = 25;
 export const CHROME_ITEM_WIDTH = (CANVAS_WIDTH - (CHROME_HORIZONTAL_PADDING * 2)) / 5;
 export const BACKEND_CHROME_ITEM_WIDTH = (BACKEND_CANVAS_WIDTH - (CHROME_HORIZONTAL_PADDING * 2)) / 10;
-export const CHROME_SINGLE_LINE_HEIGHT = 75 - (CHROME_VERTICAL_PADDING * 2);
-export const CHROME_SPACE_BETWEEN_LINES = 12.5;
+export const CHROME_SINGLE_LINE_HEIGHT = 75 - (CHROME_VERTICAL_PADDING * 2); // 25
```

Comment:
Was there previously, but where does the 75 come from? If it's a magic number anyway, why not just set this to 25? And is it intentionally the same as `CHROME_VERTICAL_PADDING` or just happen to work out that way?

Reply:
Yeah I am not a fan of all of the math here, since it doesn't seem very helpful or necessary, but I was trying to take a "light touch" approach to making this update (the 75 was preexisting in the code base).

Should I just change that line and say `25` then?

Reply:
I think the reason they did it was so that they could adjust the vertical padding in one line and have all the related constants update at one time... otherwise every time you changed that vertical padding you'd have to do a bunch of other math and update those other magic numbers. I'd be in favor of leaving it personally.

Reply:
Cool.